feature-matrix.md

```diff
@@ -18,7 +18,6 @@ The following table lists the available resource types.
 | Resource Type | Available since | Required Kubernetes and kubectl Version |
 | --- | --- | --- |
 | Virtual Machine | v0.1.0 | 1.6 |
-| Migration | v0.1.0 | 1.6 |
```

Comment:
Let's do the following:

1. Please opne a bug to reintroduce live migration
2. Stroke this entry and link it to the bug

Reply:
Done

Reply:
I'm still missing that the entrey in the table above is stroked (~...~) and a link to the bug.

Reply:
It is below the feature matrix.

Reply:
oh wrong section

Reply:
done

Reply:
thanks